palette/moreland/convert.go

```diff
@@ -0,0 +1,232 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package moreland
+
+import (
+	"image/color"
+	"math"
+)
+
+// rgb repesents a physically linear RGB color.
```

Comment:
It's been a long time since I studied color theory. Shouldn't it be "perceptually linear" here?